demo/styles/tile.css

```diff
@@ -172,6 +172,8 @@
   color: white;
```

Comment:
Why? It's just for an example page and compiled through node-sass. Should arguably be minified rather than formatted...